wandb/errors/term.py

```diff
@@ -30,17 +63,10 @@ def termlog(string='', newline=True, repeat=True, prefix=True):
     # Repeated line tracking limited to 1k messages
     if len(PRINTED_MESSAGES) < 1000:
         PRINTED_MESSAGES.add(line)
-    click.echo(line, file=sys.stderr, nl=newline)
-
-
-def termwarn(string, **kwargs):
-    string = '\n'.join(['{} {}'.format(WARN_STRING, s)
-                        for s in string.split('\n')])
-    termlog(string=string, newline=True, **kwargs)
-
-
-def termerror(string, **kwargs):
-    string = '\n'.join(['{} {}'.format(ERROR_STRING, s)
-                        for s in string.split('\n')])
-    termlog(string=string, newline=True, **kwargs)
+    if silent:
+        util.mkdir_exists_ok(os.path.dirname(util.get_log_file_path()))
+        with open(util.get_log_file_path(), 'w') as log:
```

Comment:
I think `settings.log_user` is what you want here.  maybe capture that with termsetup()

Reply:
Won't that change the logs from going to `debug-internal.log` to `debug.log`? Just confirming that's what we want.

I copied this code as is from the original PR that introduced `WANDB_SILENT`: https://github.com/wandb/client/commit/245d53bfcb0dddd9036f49eb6a5834e02b77415b

Reply:
I see that PR is super old. Will fix this.

Reply:
yes, debug.log is where logging from the user process goes,  which i think is what we want.